src/main/java/picard/arrays/CollectArraysVariantCallingMetrics.java

```diff
@@ -43,10 +43,12 @@
                     "<h4>Usage example:</h4>" +
                     "<pre>" +
                     "java -jar picard.jar CollectArraysVariantCallingMetrics \\<br />" +
+                    // TODO - ADD example
```

Comment:
I don’t think you need to update the example. I would either add an example or remove the comment. Adding a TODO should be avoided if possible.